app/src/main/java/io/apicurio/registry/rest/RegistryExceptionMapper.java

```diff
@@ -60,7 +60,7 @@ public Response toResponse(RegistryException exception) {
         if (exception instanceof NotFoundException) {
             code = HttpURLConnection.HTTP_NOT_FOUND;
         }
-        if (exception instanceof RulesException) {
+        if (exception instanceof RuleViolationException) {
```

Comment:
Could the `instanceof` here be replaced with a `getCode` method the exception classes would implement?

Reply:
Yeah, we discussed this already -- no strong feeling though. :-)